authority-portal-backend/authority-portal-quarkus/src/test/kotlin/de/sovity/authorityportal/web/tests/services/FirstUserRegistrationTest.kt

```diff
@@ -93,10 +93,9 @@ class FirstUserRegistrationTest {
         useMockNow(now)
 
         doNothing().whenever(keycloakService).sendInvitationEmail(any())
-        doNothing().whenever(keycloakService).createOrganization(any())
         doNothing().whenever(keycloakService).joinOrganization(any(), any(), any())
-        whenever(keycloakService.createUser(
-            eq(request.userEmail), eq(request.userFirstName), eq(request.userLastName), eq(request.userPassword)
+        whenever(keycloakService.createKeycloakUserAndOrganization(
+            any(), eq(request.userEmail), eq(request.userFirstName), eq(request.userLastName), any(), eq(request.userPassword)
```

Comment:
I have something against this approach of creating multiple entities at once in one function. I think it violates the low-level version of SRP (can't find it right now, but basically some module-level rules can be applied to class members as well, since sometimes they're generic enough). Things can go wrong during recourse creation, and it would be much easier to track where exactly the exception was thrown if the resource creation process was separated.
![image](https://github.com/user-attachments/assets/0b4245d1-b7f3-4bd0-bd50-615e68533fc8)

I'm not going to die on this hill in this case, but since it's KC, I'd split it to be more careful

Reply:
It makes sense if a user without organisation breaks your application. In CP this is possible, so we've separated these functions. So it's fine for me because it emphasises that a user can only exist within an organisation.